errors.go

```diff
@@ -0,0 +1,7 @@
+package tableflip
+
+type ErrNotSupported struct{}
```

Comment:
I think it's enough to make this
```
var ErrNotSupported = errors.New("tableflip: platform does not support graceful restart")
```

Reply:
Global variables unfortunately can easily be changed.

I would suggest something like this: https://dave.cheney.net/2016/04/07/constant-errors

Reply:
That's true, however I've never found this to be a problem really. My main intention is that the error works with errors.Is like in https://blog.golang.org/go1.13-errors. Up to @kohenkatz.

Reply:
The benefit of doing it the way @sagikazarmark describes is that you can add [Dave Cheney's other error handling ideas](https://dave.cheney.net/2014/12/24/inspecting-errors), but this project isn't using those for its existing errors, so I don't think it really matters either way.

Since I'm making changes anyway, I'm going to change it as originally requested just to be consistent with errors already in the project.